frame/collective/src/weights.rs

```diff
@@ -60,195 +60,253 @@ pub trait WeightInfo {
 /// Weights for pallet_collective using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	fn set_members(m: u32, n: u32, p: u32, ) -> Weight {
+	// Storage: Instance1Collective Prime (r:0 w:1)
+	// Storage: Instance1Collective Voting (r:100 w:100)
+	// Storage: Instance1Collective Proposals (r:1 w:0)
+	// Storage: Instance1Collective Members (r:1 w:1)
```

Comment:
hmm, i thought this `Instance1*` format went away?

Reply:
collective is still using old macro https://github.com/paritytech/substrate/pull/9115